_data/remote.yml

```diff
@@ -52,8 +52,11 @@ websites:
     img: joinme.png
     tfa:
       - sms
+      - email
       - totp
-    doc: https://secure.logmein.com/welcome/webhelp/EN/Pro/LogMeIn/c_common_Security_TwoStep.html
+      - proprietary
+    doc: https://secure.logmein.com/welcome/webhelp/EN/Pro/LogMeIn/c_Common_Account_Security.html
```

Comment:
Isn't https://support.logmeininc.com/pro/help#logmein-t-login-policy-set-twofactor-ownaccount better?

Reply:
Yeah, I just adjusted the links to point to the same page - but your suggestion is actually more specific.